app/graphql/types/invoices/object.rb

```diff
@@ -40,6 +40,8 @@ class Object < Types::BaseObject
       field :payment_due_date, GraphQL::Types::ISO8601Date, null: false
       field :payment_overdue, Boolean, null: false
 
+      field :all_charges_have_fees, Boolean, null: false, method: :all_charges_have_fees?
```

Comment:
Is this for the frontend to know if they should show/hide the "Add" button?

Reply:
Yes exactly! @julienbourdeau